tests/e2e-tests/specs/backend/checkout.test.js

```diff
@@ -4,17 +4,24 @@
 import {
 	insertBlock,
 	getAllBlocks,
+	openDocumentSettingsSidebar,
 	switchUserToAdmin,
 } from '@wordpress/e2e-test-utils';
-
-import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+import {
+	findLabelWithText,
+	visitBlockPage,
+} from '@woocommerce/blocks-test-utils';
 
 const block = {
 	name: 'Checkout',
 	slug: 'woocommerce/checkout',
 	class: '.wc-block-checkout',
 };
 
+const closeInserter = async () => {
```

Comment:
This has been extracted into a util in #3057. I left it as-is here and I can refactor it to use the util once it's merged.